drivers/SmartThings/harman-luxury/search-parameters.yml

```diff
@@ -0,0 +1,2 @@
+mdns:
+  - service: "_sue-st._tcp"
```

Comment:
Is this the correct service name? It's not the same one we saw in test tools, so just wanted to make sure this was accurate.

Reply:
Yes. We changed it one of our latest firmware releases. When you test on the L42ms, please ensure you are testing on latest StreamSDK software. Latest release currently is 1793, but we will make a new release once I covered the changes.

Reason for the change was to avoid discovery of unsupported devices